core/src/main/java/org/sonarsource/sonarlint/core/SonarLintClientImpl.java

```diff
@@ -93,17 +96,17 @@ private void checkStarted() {
   }
 
   private static RuntimeException handleException(RuntimeException t) {
-    if (LOG.isDebugEnabled()) {
-      return t;
-    }
-
+    StringWriter errors = new StringWriter();
+    t.printStackTrace(new PrintWriter(errors));
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1148)